addons/wardrobe/functions/fnc_getAllWardrobeItems.sqf

```diff
@@ -0,0 +1,25 @@
+#include "../script_component.hpp"
+
+/*
+ * Author: OverlordZorn
+ * Debug - Function to retrieve all existing Wardrobe items.
+ *
+ * Arguments:
+ * 0: Return as Config? <BOOL> (default: false)
+ *
+ * Return Value:
+ * Array Wardobe Items, ether as Classname or as Config <ARRAY>
+ *
+ * Example:
+ * [true] call ace_wardrobe_fnc_getAllWardrobeItems;
+ * [this, flatten ([] call ace_wardrobe_fnc_getAllWardrobeItems)] call ace_arsenal_fnc_initBox;
+ *
+ * Public: Yes
+ */
+
+
+params [["_asConfig", false, [true]]];
+
+private _return = flatten ( ["CfgWeapons", "CfgGlasses"] apply { ( QUOTE([_x] call FUNC(isModifiable)) configClasses (configFile >> _x) ) } );
+
+if (_asConfig) then { _return } else { _return apply { configName _x } }
```

Comment:
I haven't looked at this in detail, but imo processing such as this should be done where the function is being called, not within this function.

Reply:
My main gripe with this function is:

`if (_asConfig) then { _return } else { _return apply { configName _x } }`

This can be easily done on the user's end - feels like a waste of lines and performance.

---

> I use this during debugging when making the config compats.

Then this function shouldn't be public and should only be part of a script, like you've done with `compareContainerMaxLoad.sqf`, and you can ignore the first part of this comment, as it's only executed very rarely.